pyprof/parse/nvvp.py

```diff
@@ -90,40 +90,23 @@ def createMarkerTable(self):
         self.db.execute('CREATE INDEX end_index ON marker (endTime)')
         self.db.execute('CREATE INDEX id_index ON marker (id)')
 
-    def getCPUInfo(self, corrId):
-        """
-		Given the correlation id, get CPU start, end, thread id, process id.
-		The information can be in the runtime table or the driver table.
-		"""
-
-        #First look in the runtime table
-        cmd = "select start,end,processId,threadId from {} where correlationId={}".format(self.runtimeT, corrId)
-        result = self.db.select(cmd)
-        assert (len(result) <= 1)
-
-        if (len(result) == 0):
-            #Look in the driver table
-            cmd = "select start,end,processId,threadId from {} where correlationId={}".format(self.driverT, corrId)
-            result = self.db.select(cmd)
-
-        assert (len(result) == 1)
-        info = result[0]
-        start = info['start']
-        end = info['end']
-        pid = info['processId']
-        tid = info['threadId']
-        tid = tid & 0xffffffff  #convert to unsigned
-        objId = self.encode_object_id(pid, tid)
-        assert (end > start)
-        return [start, end, pid, tid, objId]
-
     def getKernelInfo(self):
         """
 		Get GPU kernel info
 		"""
-        cmd = "select name,correlationId,start,end,deviceId,streamId,gridX,gridY,gridZ,blockX,blockY,blockZ from {}".format(
-            self.kernelT
-        )
+        cmd = ("SELECT "
+              "name AS kNameId, "
+              "strings.value as name, "
+              "runtime.start as rStart, "
+              "runtime.end as rEnd, "
```

Comment:
I think you will have to coalesce `rStart` and `rEnd` as well.

Reply:
Yes, good catch. Running with bert+jit caught this.